PhpAmqpLib/Channel/AMQPChannel.php

```diff
@@ -476,11 +476,12 @@ protected function exchange_bind_ok($reader)
      * @param string $destination
      * @param string $source
      * @param string $routing_key
+     * @param bool $nowait
      * @param array $arguments
      * @param int $ticket
      * @return mixed
      */
-    public function exchange_unbind($destination, $source, $routing_key = '', $arguments = null, $ticket = null)
+    public function exchange_unbind($destination, $source, $routing_key = '', $nowait = false, $arguments = null, $ticket = null)
```

Comment:
For the sake of backward compatibility why not put the new optional parameter at the end of the list?